context/config.go

```diff
@@ -55,11 +55,12 @@ type Context struct {
 // AnkhConfig defines the shape of the ~/.ankh/config file used for global
 // configuration options
 type AnkhConfig struct {
-	SupportedEnvironments     []string           `yaml:"supported-environments"`
-	SupportedResourceProfiles []string           `yaml:"supported-resource-profiles"`
-	CurrentContextName        string             `yaml:"current-context"`
-	CurrentContext            Context            `yaml:"-"` // private, filled in by init code. The `-` instructs the yaml lib to not look for this field
-	Contexts                  map[string]Context `yaml:"contexts"`
+	SupportedEnvironments      []string           `yaml:"supported-environments"`
+	SupportedResourceProfiles  []string           `yaml:"supported-resource-profiles"`
+	SupportedImageRepositories []string           `yaml:"supported-image-repositories"`
```

Comment:
let's not enforce image repositories in `lint` - that tooling can live elsewhere. the main reason for this is because it adds another layer of configuration that i'm not convinced we want to live here.